core/silentlogger/module.go

```diff
@@ -0,0 +1,71 @@
+package silentlogger
+
+import (
+	"context"
+
+	"flamingo.me/dingo"
+
+	"flamingo.me/flamingo/v3/framework/flamingo"
+)
+
+type (
+	// Module for Silent ZAP logging
+	Module struct{}
+
+	shutdownEventSubscriber struct {
+		logger flamingo.Logger
+	}
+)
+
+// Configure the ZAP logger as flamingo.Logger (in JSON mode kibana compatible)
+func (m *Module) Configure(injector *dingo.Injector) {
+	registry := new(LoggingContextRegistry)
+
+	injector.Bind(registry).AsEagerSingleton()
+
+	injector.Bind(new(flamingo.Logger)).ToProvider(getSilentLogger)
```

Comment:
`getSilentLogger` is quite a "heavy" implementation, can this be a performance issue if we call the factory for every logger instance?

Reply:
I guess no really big differences, all info is getting cleared every request end. So it either lives in one instance, or spread across different instances. By itself it's not heavy in my opinion